website/docs/help-and-support/troubleshooting-guide/deployment-errors.mdx

```diff
@@ -184,12 +193,54 @@ Follow the steps below to downgrade to v1.8.15:
       container_name: appsmith
     ```
     </TabItem>
+
   </Tabs>
 
+2. Run the restart command:
+
+```bash
+docker-compose up -d
+```
+
+Your server starts with version 1.8.15, which makes all the necessary schema changes to your Appsmith instance.
+
+## Unable to build Kubernetes objects from current release
 
-  2. Run the restart command: 
+This issue occurs while upgrading your Appsmith Business or Enterprise Helm charts from version `3.0.4` to `3.0.7` and results in an upgrade failure.
+
+#### Error message
 
+<Message
+  messageContainerClassName="error"
+  messageContent="UPGRADE FAILED: unable to build kubernetes objects from current release manifest: resource mapping not found for name: '<chart-name>' namespace: '<namespace>' from '': no matches for kind 'HorizontalPodAutoscaler' in version 'autoscaling/v2beta1' ensure CRDs are installed first"
+></Message>
+
+#### Cause
+
+The issue happens because of how Helm handles the upgrade:
```

Comment:
let's write this as a para as Goutham shared because point 2 is a result of point 1. there's no need for us to split it here as there is just one cause.